Gemfile.lock

```diff
@@ -305,7 +307,7 @@ DEPENDENCIES
   webmock (~> 3.0)
 
 RUBY VERSION
-   ruby 2.4.1p111
+  ruby 2.4.1p111
```

Comment:
Why was the spacing changed here?

Reply:
Just fixed the indentation, that's it.

Reply:
Oh, I see.

Normally indentation fixes are good, but this is a generated file–it's a bad plan to edit it manually. If you do, next time anyone installs runs a `bundle` command, this will just change back. The end result is that it clogs up the code diff with useless information.

Could you change the spacing back to the default here and in L313?